txrelayer/txrelayer.go

```diff
@@ -187,9 +187,15 @@ func (t *TxRelayerImpl) sendTransactionLocked(txn *ethgo.Transaction, key ethgo.
 	}
 
 	if t.writer != nil {
-		_, _ = t.writer.Write([]byte(
-			fmt.Sprintf("[TxRelayer.SendTransaction]\nFrom = %s \nGas = %d \nGas Price = %d\n",
-				txn.From, txn.Gas, txn.GasPrice)))
+		msg := fmt.Sprintf("[TxRelayer.SendTransaction]\nFrom = %s\nGas = %d\nGas Price = %d\n",
```

Comment:
why not if else? this way we have additional  `fmt.Sprintf(` which is not needed

Reply:
Just wanted to spare a couple of lines (if/else block and variable declaration)... Fixed https://github.com/0xPolygon/polygon-edge/pull/1936/commits/6c64f322225a07faadc44ecc389d7df6e6f9d742